Source/WebCore/animation/DocumentTimeline.cpp

```diff
@@ -196,7 +183,7 @@ DocumentTimeline::ShouldUpdateAnimationsAndSendEvents DocumentTimeline::document
     bool wasAnimationResolutionScheduled = std::exchange(m_animationResolutionScheduled, false);
 
     if (!wasAnimationResolutionScheduled || animationsAreSuspended() || !shouldRunUpdateAnimationsAndSendEventsIgnoringSuspensionState())
-        return DocumentTimeline::ShouldUpdateAnimationsAndSendEvents::No;
+        return AnimationTimeline::ShouldUpdateAnimationsAndSendEvents::No;
```

Comment:
This line is right but the method signature should probably be changed from `DocumentTimeline::ShouldUpdateAnimationsAndSendEvents` to `AnimationTimeline::ShouldUpdateAnimationsAndSendEvents`.

Reply:
Fixed.